app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -834,6 +836,21 @@ class PipelineSampleReads extends React.Component {
           </div>
         </div>
       );
+
+    let show_amr = this.amr != null;
+    let amr_tab = show_amr ? (
+      <li className="tab">
+        <a href="#amr" className="">
+          Antimicrobial Resistance Report
```

Comment:
This is a bit wordy. Change it to just "Antimicrobial Resistance"

Reply:
done